index.js

```diff
@@ -244,9 +277,16 @@ function getGermanCars(/* code here */) {
  *   return num * 2
  * }
 */
-const sum = null; // code here!
-const addFive = null; // code here!
-const argTimesTwo = null; // code here!
+const sum = (a, b) => {
+   a + b
```

Comment:
When using an arrow function, if you are enclosing your logic within `{ }`, don't forget to use `return`. 
You can either do it this way:
```
const sum = (a, b) => a + b
```

Or this way:

```
const sum = (a, b) => {
  return a + b
}
```